Docs: Break The Monolith Updates (#440)
amplication/docs: docs/premium-features/break-the-monolith.md

### Service Filter

On the upper left-side of the Architecture tab is the filter component and the associated search bar.

Each individual colored button with an icon represents one of your services. _Hover_ over the button to see the name of your service. _Click_ the button to hide/show the service.

The associated search field helps you search through your services by name.

The filter is useful if you only need to break specific services.

### Drag-and-Drop Interface Management Buttons

Directly below the Service Filter are various buttons to help you manage the drag-and-drop interface.

_Hover_ over each button to see the name of what the button does.

1. **Toggle Field List** toggles the associated field names for all the entities in your service.
2. **Reset Layout** resets the Zoom level back to normal.
3. **Zoom In** increases the Zoom level of the interface.
4. **Zoom Out** decreases the Zoom level of the interface.

## Redesign Mode

Redesign mode allows you to restructure services by facilitating the movement of entities between them using our drag-and-drop interface.

Click on the **Redesign** button to enter Redesign Mode and select a service to redesign.

:::note

> You can currently redesign only **one** service at a time.

Reviewer: ```suggestion
You can redesign only **one** service at a time.
```

:::

You can move entities between services in two ways:

1. Drag and drop entities into the appropriate service.

> 2. Select multiple entities using the dedicated menu and move them in bulk to create micro-services.

Reviewer: ```suggestion
2. Select multiple entities using the dedicated menu or the command/control button and move them in bulk to create micro-services.
```

![Amplication's Break The Monolith Redesign Mode](./assets/break-the-monolith-redesign-mode.png)

Redesign mode introduces a few more buttons to help you while redesigning your service.

1. **Add New Service** creates a new service so that you can drag-and-drop entities between 

> 2. **Fetch Updates From Server** gets any updates from the server. This is useful in case updates were made during the monolith update process.

Reviewer: ```suggestion
2. **Fetch Updates From Server** gets any updates from the server. This is useful in case updates were made during the monolith redesign process.
```

> 3. **Discard Changes** discards all changes and brings you back into Overview Mode. 

Reviewer: ```suggestion
3. **Discard Changes** discards all the Redesign changes and brings you back into Overview Mode. 
```

### Step 4: Manage and Apply Changes
-->

> <!--

Reviewer: I think this "future" section should be at the end of the page. WDYT?

To get started with your Prisma schema upload, visit our [Import Prisma Schema](/how-to/import-prisma-schema) documentation for step-by-step instructions.
-->

## Architecture Change Confirmation

> Once you're satisfied with the changes, use the **Apply Plan** button to update your project's architecture.

Reviewer: I think the screenshot should appear after this line, and maybe we need to elaborate a little regarding the screen displayed

> After applying changes in the Architecture tab, the "Pending Changes" section of your dashboard will reflect those updates. If everything looks good, click on the **Commit Changes & Build** button and Amplication will create a new PR to your git provider with your changes.

Reviewer: ```suggestion
After applying changes in the Architecture tab, the "Pending Changes" section of your dashboard will reflect those updates. If everything looks good, click on the **Commit Changes & Build** button and Amplication will automatically create a new PR to your git provider containing your changes towards a micro-services architecture.
```

> :::note

Reviewer: I think this node should be displayed earlier in the page (I think that right before the "Architecture Change Confirmation" part
